### DISCUSSION TITLE
Connect metamask on safari mobile

### DISCUSSION THREAD

**Discussion by hoangman1108** [rainbow-me/rainbowkit#2273] [Q&A] [answered]
Can I define 1 button to connect metamask only on safari mobile? and how do I do it? I don't want to open Rainbowkit modal because I only want to connect Metamask.

**Comment by DanielSinclair** [comment, marked as answer]
@hoangman1108 We have a `WalletButton` API for this purpose. It's often best to serve multiple buttons for popular wallets, or to just rely on the `ConnectButton` so that all users can connect. https://www.rainbowkit.com/docs/wallet-button
